src/verbose.hpp

```diff
@@ -0,0 +1,23 @@
+//
+//  verbose.hpp
+//
+// Some helper declarations for verbose messages.
+
+#ifndef verbose_h
+#define verbose_h
+#include <string>
+
+struct Verbose {
```

Comment:
The question is if this really needs to be a struct, since it is just a collection of helper functions and static variables. Maybe convert to a namespace with the same name.

Reply:
You are right, namespace is better than struct here.